productivity-suite/app/fragments/todos/src/App.tsx

```diff
@@ -63,7 +63,7 @@ const App: React.FC<{
   }
 
   return (
-    <div className="todo-mvc-wrapper">
+    <div id="todos-fragment-root" className="todo-mvc-wrapper">
```

Comment:
is this the right fix? 😮 

this is the bit that uses this id:
![Screenshot 2022-10-27 at 11 32 07](https://user-images.githubusercontent.com/61631103/198262100-6c5b38e7-7897-4fad-a487-fd94a3b5d842.png)

so basically you're creating the `App` and inside it you create this div with the `todos-fragment-root` id and then you look for this div in order to know where to create/hydrate the application (which is the one creating the div in the first place)?

am I missing something? 🤔

Reply:
also locally the app does seem to be complaining about it?
![Screenshot 2022-10-27 at 11 37 15](https://user-images.githubusercontent.com/61631103/198263145-163c74df-a6fb-4539-9bb3-ec28573f0e54.png)

Reply:
Doh! I really am not doing well here, eh?
I'll just unravel most of these changes...

Reply:
ah no worries, it happens, I guess you could have missed the console log and react just shows the thing nicely anyways 😅